org.adempiere.base/src/org/compiere/print/ServerReportCtl.java

```diff
@@ -289,6 +289,7 @@ static public boolean startStandardReport (ProcessInfo pi)
 			MQuery query = MQuery.get (ctx, pi.getAD_PInstance_ID(), TableName);
 			PrintInfo info = new PrintInfo(pi);
 			re = new ReportEngine(ctx, format, query, info, pi.isSummary(), pi.getTransactionName());
+			re.setPrintFormat((MPrintFormat) o);

```

Comment:
I don't get it - isn't the format argument to ReportEngine constructor above is already doing that ( "MPrintFormat format = (MPrintFormat)o;" at line 287) ?

Reply:
> I don't get it - isn't the format argument to ReportEngine constructor above is already doing that ( "MPrintFormat format = (MPrintFormat)o;" at line 287) ?
You are right, I'll check again, maybe the problem is in the ReportEngine itself

Reply:
It was probably something bad in my environment. I was not able to reproduce using master branch. 

If you take a closer look at ReportEngine constructor, you see the PrintFormat is set directly to a variable. The method setPrintFormat does other things besides setting the field, but I spend all day and could not reproduce the random problem using the master branch. I will re-open this if It happens again and I figure it out how to reproduce properly.